<?xml version="1.0"?>
<launch> 
    <arg name="agent"             default="agent1"/>
    <arg name="scale_linear"      default="1.2"/>
    <arg name="scale_angular"     default="1"/>
    <arg name="dev"               default="/dev/input/js0"/>

    <!-- <node name="hamster_teleop_node" pkg="hamster_teleop" exec="hamster_teleop_node" namespace="$(var agent)" output="screen">
        <param name="dev"              value="$(var dev)" />
        <param name="scale_linear"     value="$(var scale_linear)"/>
        <param name="scale_angular"    value="$(var scale_angular)"/>
        <remap from="/ackermann_cmd"   to="/$(var agent)/ackermann_cmd"/>
    </node> -->
 
</launch>
